etc/7z.profile

```diff
@@ -17,6 +17,7 @@ notv
 novideo
 shell none
 tracelog
+caps.drop all
```

Comment:
are some capabilities not required for setting certain flags, file ownership etc.?

Reply:
removed caps.drop all